packages/mapping/ol/mapmodule/bundle.js

```diff
@@ -144,6 +144,14 @@ Oskari.clazz.define(
                     'type': 'text/css',
                     'src': '../../../../bundles/mapping/mapmodule/resources/scss/indexmap.ol.scss'
                 },
+                // AnnouncementsPlugin
+                {
+                    'type': 'text/javascript',
+                    'src': '../../../../bundles/mapping/mapmodule/plugin/announcements/AnnouncementsPlugin.js'
+                }, {
+                    'type': 'text/css',
+                    'src': '../../../../bundles/mapping/mapmodule/resources/scss/announcementsplugin.ol.scss'
+                },
```

Comment:
This should be part of `announcements` bundle or something but not part of mapmodule.